### ISSUE TITLE
CLI: Config issues with pull

### ISSUE THREAD

**Issue by josephjclark** [OpenFn/kit#712]
I've been doing some testing with `openfn pull` and seeing some issues

* Env vars are not loaded into the command 
* Can't specify endpoint, apikey through CLI args
* I can't work out how to set the path to pull to (why can't I `openfn pull <id> /repo/openfn/projects/my-project`)
* Paths in config.json seem to be relative to the CLI's working dir, not the config.json itself. I suspect deploy is the same.

Basically the first pull experience seems way too complex

**Comment by josephjclark** [comment]
This is sort of resolved by #714. I'm going to close the issue for now - maybe we'll look at the other bits later.
